src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
-export interface Figure {
-
+export enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
 }
 
-export class Triangle {
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shapes;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = Shapes.Triangle;
+
+  constructor(
+    public color: Color,
+
+    public a: number,
+
+    public b: number,
+
+    public c: number,
```

Comment:
Why do you need these page breaks? The same in other constructors.
```suggestion
  constructor(
    public color: Color,
    public a: number,
    public b: number,
    public c: number,
```